### DISCUSSION TITLE
Can model the CDM group for DMRS (pilot)?

### DISCUSSION THREAD

**Discussion by Mio-TW** [NVlabs/sionna#330] [Q&A] [answered]
Dear Sionna Team,

Can Sionna's pilot (DMRS) model the CDM group? 
If so, how can it be used? 
Thank you so much.

**Comment by SebastianCa** [comment, marked as answer]
Hi,
yes, you can configure different CDM groups in the `PUSCHTransmitter`.
Please see [5G NR PUSCH tutorial](https://nvlabs.github.io/sionna/examples/5G_NR_PUSCH.html#Understanding-the-DMRS-Configuration) for a detailed introduction.
